### ISSUE TITLE
Error creating a ske cluster in a fresh project

### ISSUE THREAD

**Issue by h3adex** [stackitcloud/terraform-provider-stackit#471]
When creating an SKE cluster in a fresh project using Terraform, I encounter the following error during the initial provisioning:

```
│ Wait for SKE enablement: 404 Not Found, status code 404, Body: {"code":"NotFound","message":"project id
│ \"d09468fa-ff64-4319-82bf-54502b9228e8\": not found","details":""}
```

Starting the process again with `terraform apply` resolves the issue.

Expected behavior:  if ske enablement is needed it happens in the background

**Comment by GokceGK** [comment]
Hi @h3adex , 

thanks for reporting the issue. We are aware of this problem and currently working on it. Hopefully soon, we will provide a fix.

**Comment by GokceGK** [comment]
Hi @h3adex 

This was addressed and fixed in [v0.25.0](https://registry.terraform.io/providers/stackitcloud/stackit/latest)
